### PR TITLE
Bump to version 3.3.0

### PR DESCRIPTION
**Pull request by bluewavenet**
Bump to version 3.3.0
  * Add PreAuth functionality - support for Dynamic Splash Pages support, served by NDS directly [bluewavenet]
  * Documentation updates including new sections for Splash Page and PreAuth [bluewavenet]
  * Fix memory handling bug, issue #341 [mwarning] [stevo01]
  * More reworking to support eventual ipv6 operation [wanxewoj] [mwarning] 

Signed-off-by: Rob White <rob@blue-wave.net>

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,12 @@
+nodogsplash (3.3.0)
+
+  * Add PreAuth functionality - support for Dynamic Splash Pages support, served by NDS directly [bluewavenet]
+  * Documentation updates including new sections for Splash Page and PreAuth [bluewavenet]
+  * Fix memory handling bug, issue #341 [mwarning] [stevo01]
+  * More reworking to support eventual ipv6 operation [wanxewoj] [mwarning]
```
Comment (mwarning) [inline ChangeLog]: This is somewhat better:
```
* Improvements towards usable IPv6 support [wanxewoj]
```

Comment (bluewavenet) [reply ChangeLog]: Agreed

File: release_todos [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+1. conf.h - update version number and copyright eg current year
+2. Changelog - list changes
+3. debian/changelog - list changes
+4. debian/copyright - update copyright dates as required eg current year
+5. docs/source/conf.py - update copyright notice eg current year, version and release numbers
+6. "cd docs" then "make man". Copy resulting docs/build/man/nodogsplash.1 to overwrite old debian/doc/nodogsplash.1
+7. remove /docs/build directory created in step 6
+8. Push to origin
+9. Github - Create Pull Request and Merge to master
+Finally on Github proceed to "Draft New Release"
```
Comment (mwarning) [inline release_todos]: Shouldn't this be part of the docs? Note files in the root folder will become messy.

Comment (mwarning) [reply release_todos]: Also mention the release tagging via `git tag -a v3.3.0 -m "nodogsplash release 3.3.0 "` and `git push --tags`.

Comment (bluewavenet) [reply release_todos]: You have a valid argument but I feel it is important this note is left in plain sight - we both have trouble remembering all the things to do :-D (have I remembered all of them for this note? lmao)

Comment (bluewavenet) [reply release_todos]: Ha! yes indeed - the dreaded release tagging!

Comment (mwarning) [reply release_todos]: Or put it into the wiki. Otherwise we open the door for other howto and todo files.

Comment (bluewavenet) [reply release_todos]: The door is open only if we leave it open. I agree for anything else but this really does need to be in plain sight.

Comment (mwarning) [reply release_todos]: It is important to me. Please put it in the docs or the wiki.

Comment (bluewavenet) [reply release_todos]: Wherever it goes it is important that everyone doing a release sees it.
Docs is for "How to use", so how about a "Notes" folder? Then all the howto todos can go in there, keeping things tidy.

Comment (bluewavenet) [reply release_todos]: I'd forgotten there even was a wiki...
Looks like I even added something once too. Must be getting old :-D
I wonder if anyone looks at it.

Comment (bluewavenet) [reply release_todos]: OK, It is now in a Notes folder and also in the wiki ;-)

Comment (mwarning) [reply release_todos]: Ok, then we need no additional file in the root folder then.

Comment (bluewavenet) [reply release_todos]: Agreed ;-)

Comment (bluewavenet) [reply release_todos]: I have updated the wiki somewhat, needs more work.

**Pull request merged**